### ISSUE TITLE
Bug when you can't move anywhere

### ISSUE THREAD

**Issue by enochhz** [Jester565/SpyNinjaGame#28]
<img width="693" alt="screen shot 2016-11-22 at 4 17 17 pm" src="https://cloud.githubusercontent.com/assets/17519068/20547160/2c833bfc-b0cf-11e6-98b5-ed66a60a9222.png">

**Comment by Jester565** [comment]
So... it worked? I don't get it

**Comment by jtara1** [comment]
Maybe this was for our status report? The following grid print after the 'stand still command' would be nice to see.

**Comment by enochhz** [comment]
yes, you can just shoot the ninja ahead you, and then the ninja beside just kill you, it works, but I haven't test for the situation when the player don't have bullet

**Comment by jtara1** [comment]
@walkerzheng I think that's the expected behavior. Shoot ends the player turn and if there's another enemy nearby then the enemy would kill the spy.

---

I don't think this feature (stand still) is working at the moment.

```
Move Status: Moved!
[ ][ ][ ][ ][ ][ ][ ][ ][ ]   Lives: 2
[ ][R][ ][ ][R][ ][ ][R][ ]   Ammo: 1
[r][ ][ ][N][ ][N][ ][ ][N]   Invincible Turns: 0
[ ][I][ ][ ][ ][ ][ ][ ][ ]   Radar: Disabled
[ ][R][ ][ ][R][ ][ ][R][ ]
[ ][ ][ ][ ][ ][ ][ ][N][ ]
[N][ ][ ][ ][ ][ ][ ][ ][ ]
[S][b][ ][ ][R][B][ ][R][ ]
[ ][ ][N][ ][ ][ ][ ][ ][ ]

Enter a direction to move or another command
W: Up | A: Left | S: Down | D: Right
1: Shoot | 2: Debug | 3: More Options | 4: HardMode
s
Move Status: Moved!
[ ][ ][ ][ ][ ][ ][ ][ ][ ]   Lives: 2
[ ][R][ ][ ][R][ ][ ][R][ ]   Ammo: 1
[r][ ][ ][ ][N][ ][ ][N][ ]   Invincible Turns: 0
[ ][I][ ][ ][ ][N][ ][ ][ ]   Radar: Disabled
[ ][R][ ][ ][R][ ][ ][R][ ]
[ ][ ][ ][ ][ ][ ][ ][ ][ ]
[ ][ ][ ][ ][ ][ ][ ][N][ ]
[N][b][ ][ ][R][B][ ][R][ ]
[S][N][ ][ ][ ][ ][ ][ ][ ]

You cannot move... enter S to stand still or another command
1: Shoot | 2: Debug | 3: More Options | 4: HardMode
s
Move Status: Out of bounds
[ ][ ][ ][ ][ ][ ][ ][ ][ ]   Lives: 2
[ ][R][ ][ ][R][ ][ ][R][ ]   Ammo: 1
[r][ ][ ][ ][N][ ][ ][N][ ]   Invincible Turns: 0
[ ][I][ ][ ][ ][N][ ][ ][ ]   Radar: Disabled
[ ][R][ ][ ][R][ ][ ][R][ ]
[ ][ ][ ][ ][ ][ ][ ][ ][ ]
[ ][ ][ ][ ][ ][ ][ ][N][ ]
[N][b][ ][ ][R][B][ ][R][ ]
[S][N][ ][ ][ ][ ][ ][ ][ ]

You cannot move... enter S to stand still or another command
1: Shoot | 2: Debug | 3: More Options | 4: HardMode
s
Move Status: Out of bounds
[ ][ ][ ][ ][ ][ ][ ][ ][ ]   Lives: 2
[ ][R][ ][ ][R][ ][ ][R][ ]   Ammo: 1
[r][ ][ ][ ][N][ ][ ][N][ ]   Invincible Turns: 0
[ ][I][ ][ ][ ][N][ ][ ][ ]   Radar: Disabled
[ ][R][ ][ ][R][ ][ ][R][ ]
[ ][ ][ ][ ][ ][ ][ ][ ][ ]
[ ][ ][ ][ ][ ][ ][ ][N][ ]
[N][b][ ][ ][R][B][ ][R][ ]
[S][N][ ][ ][ ][ ][ ][ ][ ]

You cannot move... enter S to stand still or another command
1: Shoot | 2: Debug | 3: More Options | 4: HardMode
```

**edit**: just noticed the value of `STAND_STILL_COMMAND` is `"S"` but when the UI takes in input it calls `toLowerCase()` on the input so `"S"` would be never be read.

Fixed with commit 2abad36e1696ae0db24624cdce06b0a013bcb2f4

Should be working 

This issue was originally discussed in #19
